specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -1817,22 +2257,88 @@
       },
       "description": "The response of a list-all operation."
     },
+    "AccessPolicyAssignmentProperties": {
+      "title": "Redis Enterprise database access policy assignment properties",
+      "description": "Properties of Redis Enterprise database access policy assignment.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true,
+          "description": "Current provisioning status of the access policy assignment."
+        },
+        "accessPolicyName": {
+          "type": "string",
+          "description": "Name of access policy under specific access policy assignment. Only default policy is supported.",
+          "default": "default",
```

Comment:
We decided to have the default be "default" here right? Do we need to instead make accessPolicyName a required property and remove this default value?

Reply:
corrected it.